### DISCUSSION TITLE
프로젝트 주제 정하기

### DISCUSSION THREAD

**Discussion by myrhymetree** [goalapa/spring-cacamuca#2] [Ideas]
프로젝트 방향성
- 1순위 : 재미(개발자와 서비스 사용자 모두에게)
- 2순위 : 트래픽 활용(불특정 유저가 호기심에라도 접근할 수 있는가)
- 3순위 : 기술 활용(현재 상태에서 적용 가능한, 공부해서 적용할 수 있는 기술이 있는지 여부) 

만들어 보고 싶은 서비스
- 예전에 개발한 서비스를 리팩토링 or 추가 사항 적용
- 실시간 예약 시스템 등(like 애완견 산책 예약 시스템), 웹소켓 개발
- MBTI 활용한 심리테스트(컨텐츠 생성 자동화 활용 검토, 애드센스 적용)
- 옷 코디에 맞춰 음악 추천
- OBTI(선 심리테스트(사용자 회원가입 시 바로 할 수 있는 항목, 후순위)
 OR _취향고르기_, SNS 홍보(SNS연동, OAUTH), 채팅(웹소켓), 옷 추천, like 에이클로젯)
- 자취생활 플래너(새로 사용한 치약 사용날짜 저장 및 조회, 냉장고에 뭐 넣어놨는지 기록 등)
- 메타버스 블라인드(커뮤니티, 채팅)
- 인디음악 추천 서비스
- 영화리뷰 평점 서비스
- 백준같은 알고리즘 문제 추천 & 풀이 사이트
- 캠핑, 차박 안내 서비스
- 참여형 여행 가이드,채팅 서비스
- 자취생 필수품 랭킹 조회 서비스
- 전세계 유명관광지 과자 리뷰 사이트

자체적인 점수 계산
- 재미 3점, 트래픽 활용 2점, 기술활용 1점

~~리팩토링 0+0+1점~~
~~실시간예약 보류~~
~~심리테스트 3+2점~~
~~- 옷코디에 맞춰 음악 추천 3+2점~~
~~OBTI 3+2+1점~~
~~자취생활플래너 0+2점~~
~~메타버스 블라인드 1+2+1점~~
~~인디음악 추천 사이트 3+2+1점~~
~~영화리뷰 평점 사이트 2+2점~~
~~알고리즘 문제 추천&풀이 사이트 0+2+1점~~

23.07.03
수정안

최종결정
~~OBTI(OBTI, 인디음악 추천 사이트 中 투표, 만장일치)~~
~~ - 기획 발표시 지금 결정된 주제에 대해서 최대한 얘기해서 피드백 받을 수 있게 하기. ~~
전세계 유명관광지 과자 리뷰 사이트
(제품의 포장상태, 선물용으로 적합한지에 초점 맞추기, 실시간 알림, oauth, 관리자, 빈도 수 등을 저장하여 마케팅 정보로 활용)
리뷰는 보기 불편한 단점이 있다 -> gpt를 활용하면 요약된 정보로 재가공 할 수 있다는 장점을 활용하기 좋음.

**Comment by Oleevia01** [comment]
OBTI 아이디어 GOOOOOOOOOOOOOOD입니다👍
